test/acceptance/behave/README.md

```diff
@@ -21,15 +29,11 @@ Note: We recommend the use of virtualenv, because is an isolated working copy of
 - Both if you are using a virtual environment or not:
```

Comment:
I was not meaning changing `~/venv` to `<venv_name>`. I just meaning to use to the ` character to make the text appear in code style. Let's disscuss at skype to avoid any further missunderstanding on this.